examples/crowd-funding/README.md

```diff
@@ -0,0 +1,237 @@
+<!-- cargo-rdme start -->
+
+# Crowd-funding Example Application
+
+This example application implements crowd-funding campaigns using fungible tokens in
+the `fungible` application. This demonstrates how to compose applications together and
+how to instantiate applications where one chain has a special role.
```

Comment:
"special role" is not descriptive enough and can mean anything.

Reply:
(I didn't write that part, but:)
I think it's _supposed_ to mean anything: It's an application where there is a designated chain that has some special meaning in whatever the application's logic is. How it's special _specifically_ in the crowd-funding application is described below.